configure.ac

```diff
@@ -143,6 +143,9 @@ if test "x$zmq_militant" = "xyes"; then
     AC_DEFINE(ZMQ_ACT_MILITANT, 1, [Enable militant API assertions])
 fi
 
+# IPC is available on all platforms supported by autotools build at the moment.
```

Comment:
it used to be disabled on openvms and vxworks though?

Reply:
Do these work through autotools? I wasn't sure about that. Can you suggest how this should be put here then?

Reply:
given it was just checking the OS, you could just add new cases in the hostos switch where it's undefined:

case "${host_os}" in
      vxworks*)
..
     openvms*)

Reply:
Thanks! Will fix that later on. Need to catch my flight now.

Reply:
Ok, I added these exceptions now, hope it's fine!